<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1466</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PlotWidget" name="Image1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="PlotWidget" name="Image2"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
        <item>
         <property name="text">
          <string>MAGNITUDE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PHASE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>REAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IMAGE</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Output1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="PlotWidget" name="Output1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Output2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="PlotWidget" name="OutPut2"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 0, 255);
</string>
        </property>
        <property name="title">
         <string>MIXER OUTPUT TO:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <widget class="QComboBox" name="OUTOPTION">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
</string>
           </property>
           <item>
            <property name="text">
             <string>OUTPUT 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OUTPUT 2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Component 1:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="Comb11">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
</string>
           </property>
           <item>
            <property name="text">
             <string>Image 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Image 2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="Comb12">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
</string>
           </property>
           <item>
            <property name="text">
             <string>MAGNITUDE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PHASE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>REAL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IMAG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UNIT PAHSE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UNIT</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Component 2:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="Comb21">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
</string>
           </property>
           <item>
            <property name="text">
             <string>Image 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Image 2</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSlider" name="horizontalSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="Comb22">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
</string>
           </property>
           <item>
            <property name="text">
             <string>MAGNITUDE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PHASE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>REAL </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IMAG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UNIT PHASE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UNIT </string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>MIX</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Image 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_9">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
        <item>
         <property name="text">
          <string>MAGNITUDE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PHASE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>REAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IMAGE</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="PlotWidget" name="Image3"/>
      </item>
      <item row="1" column="1">
       <widget class="PlotWidget" name="Image4"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1466</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Image_1"/>
    <addaction name="actionImport_Image_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Image_1">
   <property name="text">
    <string>Import Image 1</string>
   </property>
  </action>
  <action name="actionImport_Image_2">
   <property name="text">
    <string>Import Image 2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
